resources/[gameplay]/gcshop/horns/horns_c.lua

```diff
@@ -391,6 +391,7 @@ hornsTable = {
 	[388] = "Kylie",
 	[389] = "Wololo",
 	[390] = "Don't Poison Your hert"
```

Comment:
Always make sure to have table values separated with a comma at the end of the previous line.